src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Set-AzsComputeQuota.ps1

```diff
@@ -59,7 +65,7 @@ function Set-AzsComputeQuota {
 
         [Parameter(Mandatory = $false)]
         [int32]
```

Comment:
This is a breaking change!  Create an alias with the old parameter and check to see if it is called, output a Warning if old parameter is being called.

Reply:
Here is an example, https://github.com/Azure/azure-powershell/blob/3d76129bbd8ac45b6787099a590ee8392f739cf8/src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsSubscriptionsQuota.ps1#L65